nomad/periodic.go

```diff
@@ -278,10 +278,10 @@ func (p *PeriodicDispatch) removeLocked(jobID structs.NamespacedID) error {
 // subsequent eval.
 func (p *PeriodicDispatch) ForceRun(namespace, jobID string) (*structs.Evaluation, error) {
 	p.l.Lock()
+	defer p.l.Unlock()
```

Comment:
I think we need to go back to the original style here, as the comment on the `createEval` function reads "This should not be called with the lock held."